pages/index.py

```diff
@@ -155,6 +162,8 @@ def slider_item(title, text, active=""):
         nonprofit = _("The Freenet Project Inc is a non-profit 501(c)(3) organization.")
         tax_deductable = _("""Donations are tax-deductible.""")
         return substitute_html(content,
+            html__slogan=slogan,
+            html__mission=mission,
```

Comment:
This is not arbitrary HTML but rather markdown content. Although this approach is valid (HTML is a superclass of Markdown), please use the `md__` prefix instead: it lets the editor of the template know that textual content is expected at this position.